packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -5185,6 +5186,10 @@ export class ContainerRuntime
 		return entry.interface as T;
 	}
 
+	public getMinVersionForCollab(): MinimumVersionForCollab {
```

Comment:
I think this should just be a property `minVersionForCollab` instead of a function. Looks like it is changed to a property in data store runtime. It would be nice to have it consistent across all layers.